src/controller/AutoCommissioner.cpp

```diff
@@ -96,26 +96,72 @@ CHIP_ERROR AutoCommissioner::SetCommissioningParameters(const CommissioningParam
     return CHIP_NO_ERROR;
 }
 
+bool AutoCommissioner::NetworkClusterUseable()
+{
+    return (mNetworkTechnology.Has(app::Clusters::NetworkCommissioning::NetworkCommissioningFeature::kWiFiNetworkInterface) &&
```

Comment:
Will this be a problem for a device on ethernet (or that otherwise gets it creds using means outside Matter)? For example, I'm not sure it makes sense for a tv casting phone app to try to fake out these network commissioning cluster features

Reply:
This part is not reached for on-network commissioning.  As in, if we discovered the device over dns-sd already, this part will not matter.